go.mod

```diff
@@ -0,0 +1,14 @@
+module sigs.k8s.io/kubetest2
+
+go 1.14
+
+require (
+	github.com/aws/aws-k8s-tester v1.0.0
```

Comment:
we should get this into a sub-module (just containing a tester?) for sure.

Reply:
you mean get a different go.mod for the eks deployer right?
i will probably do that later with the correct naming sorted out.